src/components/data-processing/StepsIndicator.jsx

```diff
@@ -27,7 +27,8 @@ const StepsIndicator = (props) => {
           dataTestClass = integrationTestConstants.classes.QC_STEP_NOT_COMPLETED;
```

Comment:
Why is this assigned apart? Why not just use `integrationTestConstants.classes.QC_STEP_NOT_COMPLETED` in `data-test-class= `

Reply:
it switches between assigning `integrationTestConstants.classes.QC_STEP_COMPLETED` and `integrationTestConstants.classes.QC_STEP_NOT_COMPLETED` depending on the state of the step and it's status. I wrote it that way because the alternative is to write it directly in `data-test-class` and it's going to be long,